### PR TITLE
Upgrading to laravel 5.3 from 5.1 with some minor fixes

### PR DESCRIPTION
**Pull request by sopnopriyo**
* Upgrading to laravel 5.3 from 5.1
* Using laravel build in authentication functionalities
* fixing some minor css issues
* protecting some routes for admin access only

### CODE REVIEW

**Comment by zeke8402** [comment]
Wow, much appreciated!  

I won't be available to look at it this weekend but I'll verify it all works some time next week and pull it in.

Thanks again!

**Comment by sopnopriyo** [comment]
Thank you very much !

File: app/Http/Controllers/AdminController.php [hunk 1]
```diff
@@ -80,4 +80,24 @@ public function anySetTime()
...
 
+  public function sendsms(Request $request){
+
+    $input = Input::all();
+
+    try{
+      Nexmo::message()->send([
+           'to' => '6'.$input['number'],
+           'from' => 'Saimedic Clinic',
+           'text' => 'Updates from Saimedic Clinic : '.$input['text']
```
Comment (zeke8402) [inline app/Http/Controllers/AdminController.php]: Sorry, I'm not going to pull in changes that include content specific to your business. You'll have to find a way to abstract it if you want it pulled in.

File: config/app.php [hunk 2]
```diff
@@ -4,6 +4,31 @@
...
     |--------------------------------------------------------------------------
+    | Application Name
+    |--------------------------------------------------------------------------
+    |
+    | This value is the name of your application. This value is used when the
+    | framework needs to place the application's name in a notification or
+    | any other location as required by the application or its packages.
+    */
+
+    'name' => 'Mediviron',
```
Comment (zeke8402) [inline config/app.php]: Another instance of business-specific variables. This should go in the .env file.

**Comment by zeke8402** [comment]
My apologies for the delay, this project is not a top priority anymore and so the amount of time I can dedicate to it is scarce.  

I can already tell from a brief review that there is a lot of content included in this merge that is more than just a framework upgrade, it also includes several files and logic that is specific to whatever business or company you're working for or with. I want this to be as bare-bones as possible for future cloners, so they can customize it to their will in the future.  

I'm not going to reject the pull, instead I will take some time when I can to review it more in-depth and suggest things to remove that would be acceptable to merge.  

Thanks again for your work!

**Comment by zeke8402** [comment]
Hi sopnopriyo, it looks like from your latest commits that you are hard at work for a client, that's great! However, I will not be able to pull in your changes to master as they contain client specific data (changing values to fit the needs of a hospital) and adding credits to yourself. While I totally support this, I will not be making it part of the original repository.

I will upgrade to 5.3 myself in the future, please continue to work on your own fork. Thanks!

**Comment by Hope777** [comment]
HI Zeke, This might seem out of the blue but wondering if you had been able to update it to either laravel 5.3 or 5.4 at all? Would save me some time if so as would like to use this for a university project! Great work and thanks!

Best Regards, 

Victor

**Pull request closed without merging**